libvast/vast/system/query_status.hpp

```diff
@@ -26,6 +26,8 @@ struct query_status {
   size_t expected = 0;      ///< Expected ID sets from INDEX.
   size_t scheduled = 0;     ///< Scheduled partitions (ID sets) at INDEX.
   size_t received = 0;      ///< Received ID sets from INDEX.
+  size_t lookups_issued = 0;
```

Comment:
All other member variables are documented. Mind adding documentation to the two new ones?